src/Connector.php

```diff
@@ -62,6 +62,50 @@ public function get(string $urlPath)
         return $this->parseResponse($response);
     }
 
+    /**
+     * Convert the data to JSON and post it to a URL.
+     *
+     * @param string $urlPath The URL to post to.
+     * @param array  $data    An array with data to post to the API.
+     *
+     * @throws \Exonet\Api\Exceptions\ExonetApiException If there was a problem with the request.
+     *
+     * @return ApiResource|ApiResourceIdentifier|ApiResourceSet The response from the API, converted to resources.
+     */
+    public function post(string $urlPath, array $data)
+    {
+        $this->apiClient->log()->debug('Sending [POST] request', ['url' => $this->apiClient->getApiUrl().$urlPath]);
+
+        $request = new Request(
+            'POST',
+            $this->apiClient->getApiUrl().$urlPath,
+            $this->getDefaultHeaders(),
+            json_encode($data)
+        );
+
+        $response = $this->httpClient->send($request);
+
+        return $this->parseResponse($response);
+    }
+
+    /**
+     * Make a DELETE call to the API.
+     *
+     * @param string $urlPath The url to make the DELETE request to.
+     */
+    public function delete(string $urlPath)
+    {
+        $this->apiClient->log()->debug('Sending [DELETE] request', ['url' => $this->apiClient->getApiUrl().$urlPath]);
```

Comment:
You could store `$url = $this->apiClient->getApiUrl().$urlPath`, because you are using this 2 times in this method. 
Same goes for the `POST` method above.

Maybe even better would be to provide the `$urlPath` as parameter to `getApiUrl`. This way you don't have to concat the output of `getApiUrl` and `$urlPath` on multiple places in you app.